src/ComServerHelpers/BaseActivationFactory.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.Versioning;
+
+namespace ComServerHelpers;
+
+/// <summary>
+/// Base for a WinRT Activation Factory for a .NET type.
+/// </summary>
+[SupportedOSPlatform("windows8.0")]
+public abstract class BaseActivationFactory
+{
+    /// <inheritdoc/>
+    public abstract object ActivateInstance();
+
+    /// <summary>
+    /// Gets the Activatable Class ID.
+    /// </summary>
+    public abstract string ActivatableClassId
+    {
+        get;
+    }
+
+    /// <summary>
+    /// Occurs when a new instance is created.
+    /// </summary>
+    public event EventHandler<InstanceCreatedEventArgs>? InstanceCreated;
```

Comment:
Sort member types (eg. fields before properties, methods after properties, sorted by accessibility, etc.).
Same applies to all other files too.

Reply:
Done:)